bears/r/FormatRBear.py

```diff
@@ -56,25 +56,33 @@ def create_arguments(filename, file, config_file,
             Maximum number of characters for an expression.
 
             Example: If ``20`` then
-            ``1 + 1 + 1 + 1 + 1 + 1 + 1`` changes to
-            ``1 + 1 + 1 + 1 + 1 + 1 +``
-            ``    1``
+            '''
+            1 + 1 + 1 + 1 + 1 + 1 + 1
+            '''
+             changes to
+            '''
+            1 + 1 + 1 + 1 + 1 + 1 +
+                1
+            '''
```

Comment:
naaaaah use triple backticks ```, I just used ''' in my example because github markdown would have interpreted them and my comment would have gotten messed up :D :D